src/Umbraco.Web.UI.Client/src/utilities.js

```diff
@@ -33,8 +33,9 @@
     /**
      * Facade to angular.extend
      * Use this with Angular objects, for vanilla JS objects, use Object.assign()
+     * This is an alias as it to allow passing an unknown number of arguments
```

Comment:
@nathanwoulfe a minor issue with the wording here "as it to allow".

Reply:
That's what happens when I type before coffee. Will fix that up, and I need to double check the changes in keyboard.service.js, I think there's a bit in there that needs to change back to the original, else we're updating the default options, which we don't want.

Reply:
@bjarnef would you mind pushing a new PR from your feature branch, to fix that typo and update keyboard.service.js:

ln26:         opt   = Utilities.extend({}, defaultOpt, opt);

I can't push to the branch anymore, probably because the original PR has been merged... If you could make that change, I'll merge it straight away.